packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -127,11 +164,35 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   Offset _currentFocalPoint;
   double _initialSpan;
   double _currentSpan;
+  _LineBetweenPointers _initialLine;
+  _LineBetweenPointers _currentLine;
   Map<int, Offset> _pointerLocations;
+  /// A queue to sort pointers in order of entrance
```

Comment:
nit: make this a comment on the end of the _pointerQueue line, since it's part of a bunch of other field declarations.